code/datums/mapgen/Cavegens/LavalandGenerator.dm

```diff
@@ -8,9 +8,9 @@
 		/obj/effect/spawner/random/lavaland_mob/legion = 30,
 		/obj/effect/spawner/random/lavaland_mob/watcher = 40,
 		/mob/living/basic/mining/bileworm = 20,
+		/mob/living/basic/mining/goldgrub = 10,
```

Comment:
uhhhhh maybe actually don't alphabetize this one and keep it the way it was ordered by the weight (value)